packages-internal/test-utils/src/describeConformance.tsx

```diff
@@ -644,7 +644,7 @@ function testSlotPropsCallbackWithPropsAsOwnerState(
       const { queryByTestId } = await render(
         React.cloneElement(element, { slotProps, className: 'custom' }),
       );
-      const slotComponent = queryByTestId('custom');
+      const slotComponent = queryByTestId('custom', { exact: false });
```

Comment:
Why is this required?

Reply:
For slots that are passed to inner component like the `backdrop`, the `className` from the `ownerState` is concatenated with the outer component's `className`, so the value is not exactly `custom`.

The `{ exact: false }` still capture the `custom` class but neglect other classes from the outer component.

Reply:
But this is `queryByTestId` 🤔, not `className`

Reply:
> But this is queryByTestId 🤔, not className

The `className` is used as a value for `data-testid` field.

Here is the full test:

```js
    const slotProps = {
        [slotName]: (ownerState: Record<string, any>) => ({
          'data-testid': ownerState.className,
        }),
      };
      const { queryByTestId } = await render(
        React.cloneElement(element, { slotProps, className: 'custom' }),
      );
```